components/select/select-top-control.component.ts

```diff
@@ -245,4 +247,10 @@ export class NzSelectTopControlComponent implements OnChanges {
       this.listOfSlicedItem = listOfSlicedItem;
     }
   }
+  private getLabelToShow(text: string | null): string | null {
```

Comment:
o.nzLabel maybe string|null|TemplateRef<any>

Reply:
@vthinkxie According to the documentation and the Typescript definition `NzOptionComponent.nzLabel` and `NzSelectItemInterface.nzLabel` can only be `string|null`:

```typescript
export interface NzSelectItemInterface {
  // ...
  nzLabel: string | null;
  // ...
}
```

`NzOptionGroupComponent.nzLabel` can be a `TemplateRef`.

Reply:
Up ⬆️